pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -120,6 +125,7 @@ type monitor struct {
 }
 
 type monitors map[schema.GroupVersionResource]*monitor
+type resourceSet map[schema.GroupVersionResource]struct{}
```

Comment:
is it simpler to use bool instead of struct{}?

Reply:
I am using struct{} to be consistent with the `ignoredResources` field on the GraphBuilder.

https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/garbagecollector/graph_builder.go#L113

Reply:
> and the reason ignoredResources does it that way I think is just because empty structs are smaller than bools.

+1.